endpoints.go

```diff
@@ -7,6 +7,7 @@ import (
 	"strings"
 
 	"go.etcd.io/etcd/client/pkg/v3/srv"
+	"golang.org/x/exp/slices"
 )
 
 func endpointsWithLeaderAtEnd(gcfg globalConfig, statusList []epStatus) ([]string, error) {
```

Comment:
Upon reading again I realize it put the leader at the end which is no longer the case after sorting